www/cocoa-rest-client/Portfile

```diff
@@ -1,61 +1,49 @@
-
-homepage                    http://mmattozzi.github.io/cocoa-rest-client/
-
-distname                    CocoaRestClient-${version}
-extract.suffix              .dmg
-
-checksums                   rmd160  0136fcc44f38bf9df4e0270e6f6e2de6b4cdac06 \
-                            sha256  dccc638d7b8e9f793055cb1c4fc49da9f0f35e24e89f3559e53810150c77a6b1 \
-                            size    6070132
-
-pre-fetch                   {
-                            if { ${os.major} <= 15 } {
-                                    return -code error "This portrequires at least macOS Sierra (10.12)."
-                                    }
-                            }
-
-extract                     {
-                            file mkdir /tmp/${distname}
-                            system "hdiutil attach ${distpath}/${distfiles} -private -nobrowse -mountpoint /tmp/${distname}"
-                            }
-
-use_configure               no
-
-build                       {}
-
-destroot                    {
-                            copy /tmp/${distname}/CocoaRestClient.app ${destroot}${applications_dir}
-                            }
-
-post-destroot               {
-                            system "hdiutil detach /tmp/${distname}"
-                            delete /tmp/${distname}
-                            }
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mmattozzi cocoa-rest-client 1.4.5
+github.tarball_from releases
+
+categories          www
+platforms           darwin
+supported_archs     x86_64
+license             BSD
+maintainers         {@lhaeger} openmaintainer
+
+description         A free, native macOS app for testing HTTP/REST endpoints
+long_description    Test and interact with HTTP/REST resources:\
+                    - Make GET, PUT, POST, DELETE, and HEAD calls.\
+                    - Set request headers and display response headers.\
+                    - Automatically pretty print XML and JSON content.\
+                    - Quick save of request URLs, body, and headers in one convenient drawer.\
+                    - Lightweight: Low real memory usage.\
+                    - SSL Support (including untrusted certificates).\
+                    - Optionally follows HTTP redirects.\
+                    - Set HTTP Basic or Digest Auth.\
+                    - Set HTTP request body content to a raw text blob or a list of parameters.\
+                    - Upload files using HTTP multipart requests. HTTP form encoding also supported.\
+                    - Unified diff tool for comparing responses.
+
+homepage            http://mmattozzi.github.io/cocoa-rest-client/
+
+distname            CocoaRestClient-${version}
+use_dmg             yes
+
+checksums           rmd160  0136fcc44f38bf9df4e0270e6f6e2de6b4cdac06 \
+                    sha256  dccc638d7b8e9f793055cb1c4fc49da9f0f35e24e89f3559e53810150c77a6b1 \
+                    size    6070132
+
+pre-fetch {
+    if {${os.major} <= 15} {
```

Comment:
I would make this `< 16`, but that's a purely personal taste.
(My reason behind would be that 15.2 for example wouldn't pass the test; but sure, the minor version is not a part of comparison when written this way.)

Reply:
We also save a byte on the Portfile, win-win!